x/derivatives/types/expected_keepers.go

```diff
@@ -29,3 +30,8 @@ type PricefeedKeeper interface {
 	// These are used for testing TODO replace mockApp with keeper in tests to remove these
 	SetParams(sdk.Context, pftypes.Params)
 }
+
+type AccountKeeper interface {
```

Comment:
I think account keeper is not used other than GetModuleAddress and this could be removed if there's no further use case since GetModuleAddress can be replaced with authtypes.NewModuleAddress

Reply:
Okay cool